src/categories/components/CategoryCreatePage/form.tsx

```diff
@@ -1,15 +1,15 @@
 import { OutputData } from "@editorjs/editorjs";
-import { ExitFormDialogContext } from "@saleor/components/Form/ExitFormDialogProvider";
+import useExitFormDialog from "@saleor/components/Form/useExitFormDialog";
```

Comment:
Can we use named exports wherever possible? 
```suggestion
import { useExitFormDialog } from "@saleor/components/Form/useExitFormDialog";
```